include/deal.II/particles/particle_handler.h

```diff
@@ -231,6 +237,57 @@ namespace Particles
     void
     insert_particles(const std::vector<Point<spacedim>> &positions);
 
+    /**
+     * Create and insert a number of particles into the collection of particles.
+     * This function takes a list of positions and creates a set of particles
+     * at these positions, which are then distributed and added to the local
+     * particle collection of a procesor. Note that this function uses
+     * GridTools::distributed_compute_point_locations. Consequently, it can
+     * require intense communications between the processors.
+     *
+     * This function figures out what mpi process owns all points that do not
+     * fall within the locally owned part of the triangulation, it sends
+     * to that process the points passed to this function on this process,
+     * and receives from the points that fall within the locally owned cells of
+     * the triangulation from whoever owns them.
+     *
+     * In order to keep track of what mpi process recieved what points, a maps
+     * from mpi process to IndexSet is returned by the function, that contains
+     * the local indices of the points that were passed to this function on the
+     * calling mpi process, and that falls within the part of triangulation
+     * owned by this mpi process.
+     *
+     * @param[in] A vector of points that do not need to be on the local
+     * processor
+     *
+     * @param[in] A vector of vectors of bounding boxes. The bounding boxes
+     * global_bboxes[rk] describe which part of the mesh is locally owned by
+     * the mpi process with rank rk. The local description can be obtained from
+     * GridTools::compute_mesh_predicate_bounding_box, and the global one can
+     * be obtained by passing the local ones to Utilities::MPI::all_gather.
+     *
+     * @param[in] (Optional) a vector of properties associated with each
+     * local point. The size of the vector should be either zero (no
+     * properties will be transfered nor attached to the generated particles)
+     * or it should be `positions.size()*this->n_properties_per_particle()`.
+     * Notice that this function call will tranfer the properties from the
+     * local mpi process to the final mpi process that will own each of the
+     * particle, and it may therefore be communication intensive.
+     *
+     * @return A pair of maps from owner to IndexSet, that contains the local
```

Comment:
We originally returned both. However the second can be obtained by calling `second = some_to_some(first)`, and may not be used at all, so we decided to remove it later on (it can be obtained with a one liner, afterall). This part of the comment was a left over... I'm fixing it now, and improving the comment.